API_MO/conventions/SingleRoomSRIR_1.0.csv

```diff
@@ -61,7 +61,9 @@ Data.SamplingRate	48000	m	I, M	double	Sampling rate of the samples in Data.IR an
 Data.SamplingRate:Units	hertz	m		attribute	Unit of the sampling rate
 Data.Delay	0	m	IR, MR	double	Additional delay of each IR (in samples)
 RoomCornerA	[0 0 0]		IC, MC	double
+RoomCornerA:Type	cartesian			attribute
+RoomCornerA:Units	metre			attribute
 RoomCornerB	[1 2 3]		IC, MC	double
-RoomCorners:Type	cartesian			attribute
-RoomCorners:Units	metre			attribute
+RoomCornerB:Type	cartesian			attribute
+RoomCornerB:Units	metre			attribute

```

Comment:
It was intended not to have `RoomCorners:Type` and `RoomCorners:Units` `RoomCornerA:Type` instead of `RoomCornerA:Units`. However, this might not agree to the netCDF standard that does not allow stale Attributes  `RoomCorners:Units` with out having a variable  `RoomCorners`